src/background/raven.js

```diff
@@ -115,7 +115,7 @@ export function report(error, when, context) {
  * automatically, in which case this code can simply be removed.
  */
 function installUnhandledPromiseErrorHandler() {
-  window.addEventListener('unhandledrejection', function (event) {
+  globalThis.addEventListener('unhandledrejection', function (event) {
```

Comment:
To recap, `globalThis` refers to `window` in a regular web page, but whatever the "global object" is in other environments. In Manifest V3, this will be a Service Worker.